src/common/interpolation.c

```diff
@@ -107,7 +107,7 @@ static inline int clip(int i, int min, int max, enum border_mode mode)
     case BORDER_MIRROR:
       if(i < min)
       {
-        i = min - i;
+        i = 2 * min - i;
```

Comment:
It might be good to add an explaining comment here why we need `2*min` , it took me a while to understand this :-)

Reply:
Yes, I'm not sure to fully understand this.

Reply:
Added a commit with better comments. In process of doing that, realized that this is probably a much lower priority bug. See note below.